v2/tools/asoctl/internal/importing/find_child_resources.go

```diff
@@ -45,6 +45,12 @@ func createTypeToSubtypesMap() map[string]set.Set[string] {
 
 		rsrc, ok := obj.(genruntime.KubernetesResource)
 		if !ok {
+			// Skip non-resources
```

Comment:
Curious when this happens? I would've thought that every GVK pointed to a resource?